django/core/paginator.py

```diff
@@ -1,6 +1,6 @@
 import collections
+import warnings
 from math import ceil
-
 from django.utils import six
 
```

Comment:
I propose to add the blank line again to separate imports of standard libraries.